core/context.go

```diff
@@ -85,6 +85,47 @@ func (c *Context) FilterUsers(filters ...string) (filtered []User) {
 	return filtered
 }
 
+// GetRootUser gets the root user
+func (c *Context) GetRootUser() User {
```

Comment:
FYI: I know it's not because of you, so just leaving a comment: a "context" is very C-like and not a great tool to make code readable, since organization-wise it's a stash you throw anything in. A "user" package would probably be better on topic.